### DISCUSSION TITLE
how i can scan QR the the challenge

### DISCUSSION THREAD

**Discussion by Hamid210-coder** [Cyfrin/foundry-full-course-cu#220] [General]
hii all, i have just completed lesson 1, but dont know how to scan QR from laptop

**Comment by 0xtdey** [comment]
You can take a screenshot of the QR codes and go to google image search and paste it over there, google will automatically show you the website link and you can go from there. 

Let me know if you are able to do it.

**Reply by Hamid210-coder** [reply]
thank you ,
i got it, but know i dont know how to complete the challenge,i dont know solidity,i have just taken lesson 1 
im total beginner

**Reply by 0xtdey** [reply]
No worries, after you get to the page in etherscan, head over to Code and read the challenge contract then head over to write contract for answering, try for a bit and then let me know if you are able to get it

**Reply by Hamid210-coder** [reply]
sorry bro,i still cant get it ?
there are 6 files having code written on it , how can i write code,?
![Screenshot (46)](https://github.com/Cyfrin/foundry-full-course-f23/assets/135648670/12cbe0e7-e288-4238-a1d2-b066b0ce0636)

**Reply by Shiva-Sai-ssb** [reply]
Hello @Hamid210-coder, for this challenge, there is no need to write any code. Simply go to the "Write Contract" tab, connect your MetaMask wallet, and solve the challenge. You can refer to the comments in the first Solidity file for guidance on how to complete the challenge.

**Reply by 0xtdey** [reply]
@Shiva-Sai-ssb He is a beginner don't give the answer directly let him figure it out. But anyways @Hamid210-coder , this is how you would have to do it as @Shiva-Sai-ssb mentioned

**Reply by Hamid210-coder** [reply]
thank you to both of you
